database/postgres/build_list.go

```diff
@@ -159,6 +159,8 @@ func (c *client) GetRepoBuildList(r *library.Repo, filters map[string]interface{
 	err = c.Postgres.
 		Table(constants.TableBuild).
 		Where("repo_id = ?", r.GetID()).
+		Where("started < ?", before).
+		Where("started > ?", after).
```

Comment:
Yeah, I think `created` would be better personally just because it's more full representation of start time including it being in the queue, but I'm not sure about the customer use case. 

I think either would be fine, just have a preference towards created.